### PR TITLE
64-bit integers and CI tests

### PR DESCRIPTION
**Pull request by coprigent**
This updates adds Github actions for running continuous integrations tests with 64-bit integers.
It also fixes somes bugs related to data reading in binary files.

### CODE REVIEW

File: src/mmgs/inout_s.c [hunk 1]
```diff
@@ -171,7 +172,7 @@ int MMGS_loadMesh(MMG5_pMesh mesh, const char *filename) {
     }
     MMG_FREAD(&mesh->ver,MMG5_SW,1,inm);
     if(iswp) mesh->ver = MMG5_swapbin(mesh->ver);
-    while(fread(&binch,MMG5_SW,1,inm)!=0 && binch!=54 ) {
+    while(fread(&binch,4,1,inm)!=0 && binch!=54 ) {
```
Comment (Algiane) [inline src/mmgs/inout_s.c:175]: Why removing the macro here? Is it not the same?

**Review by Algiane — COMMENTED** [review]
Thanks!

File: src/mmgs/inout_s.c (hunk 1 above)
Comment (coprigent) [reply src/mmgs/inout_s.c]: This is a mistake. I reverted this change.

**Comment by Algiane** [comment]
Perfect, thanks!

**Pull request merged**